### PR TITLE
feat: add paypal api and update charge entity

### LINKED ISSUE #393: add paypal api and update charges entity
working on it 👍

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes #393

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/paypal/CreatePaypalPaymentResponse.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
+data class CreatePaypalPaymentResponse(
+        val status: Boolean,
+        val paymentId: String,
+        val error: String
+
```
Comment (dreadpool2) [inline app/src/main/java/org/fossasia/openevent/general/paypal/CreatePaypalPaymentResponse.kt]: extra line

**Comment by simarsingh24** [comment]
@nikit19 @dreadpool2 please review

**Comment by dreadpool2** [comment]
please resolve conflicts

**Comment by dreadpool2** [comment]
I am merging this as @simarsingh24 as resolved conflicts

**Pull request merged**